homeassistant/components/image/__init__.py

```diff
@@ -146,8 +151,48 @@ def image(self) -> bytes | None:
 
     async def async_image(self) -> bytes | None:
         """Return bytes of image."""
+
+        async def _async_load_image_from_url(url: str) -> Image | None:
```

Comment:
Consider making a method of this function instead so we don't need to make a copy every time this code runs.

Reply:
There's no copy as such happening because of the inner function.